modules/resolver/src/main/java/org/apache/tamaya/resolver/internal/FileResolver.java

```diff
@@ -86,7 +86,12 @@ public String evaluate(String expression) {
                 builder.append(inputLine).append("\n");
             }
 
-            return builder.toString();
+            String content = builder.toString();
+            if (content.endsWith("\n")) {
+                content = content.substring(0, conent.length() - 2);        
```

Comment:
conent? Seems like a typo to me.

Is it possible to provide/augment a test case to verify the behaviour and prevent regressions in the future. Thanks.